packages/ember-simple-auth/addon/session-stores/adaptive.js

```diff
@@ -112,28 +123,32 @@ export default Base.extend({
       this._fastboot = owner.lookup('service:fastboot');
     }
 
-    this._isLocalStorageAvailable = this.hasOwnProperty('_isLocalStorageAvailable') ? this._isLocalStorageAvailable : testLocalStorageAvailable();
-
     let store;
     if (this.get('_isLocalStorageAvailable')) {
+      const localStorage = owner.lookup('session-store:local-storage');
       const options = { key: this.get('localStorageKey') };
+
       options._isFastBoot = false;
-      store = this._createStore(LocalStorage, options);
+      localStorage.setProperties(options);
+
+      store = localStorage;
+      this.localStorage = localStorage;
```

Comment:
Is this used anywhere?

Reply:
👍  I was under the impression that we want to expose the services as properties as well.

I'll remove that next time.